examples/source/1.components/amp-script/index.html

```diff
@@ -484,5 +488,40 @@ <h2 id="translated-greeting"></h2>
         button.addEventListener('click', randomSlideDirection);
       </script>
     </div>  
+    <!--
+        ## Local storage
+        Your script has access to the Web Storage API. This lets you persist user information between browser sessions.
+        Here, we use `localStorage` to track a username.
+        If you set the username, then reload this page, the username will remain set.
+    -->
+    <div>
+      <amp-script layout="fixed-height" script="local-storage-script" height="110" class="sample">
+        <div>
+          <span>Current username: </span>
+          <span id="username-display"></span>
```

Comment:
Any reason to not use a [`b`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b) tag here for this? Seems ideally suited semantically as well:

> The **HTML Bring Attention To element** (`<b>`) is used to draw the reader's attention to the element's contents, which are not otherwise granted special importance.

```suggestion
          <b id="username-display"></b>
```

Then the style change wouldn't be needed either.